web/docs/try/quickstart.md

```diff
@@ -401,12 +402,12 @@ maximum data density.
 ### Extract data with rich expressions
 
 Finally, let's get a feel for the [expression
-language](../understand/expressions.md). VAST comes with native types for IP
+language](../understand/expressions.md). Tenzir comes with native types for IP
 addresses, subnets, timestamps, and durations. These come in handy to succinctly
 describe what you want:
 
 ```bash
-vast export json '10.10.5.0/25 && (orig_bytes > 1 Mi || duration > 30 min) | select orig_h, resp_h, orig_bytes'
+tenzirctl export json '10.10.5.0/25 && (orig_bytes > 1 Mi || duration > 30 min) | select orig_h, resp_h, orig_bytes'
```

Comment:
I'm not commenting this here on every instance now, but we should make a pass over the docs and replace all instances of `tenzirctl export` and `tenzirctl import` with `tenzir 'export | …'` and `tenzir '… | import'`, respectively.

Reply:
Yep, I have a todo item that tells me to go over all uses of `tenzirctl`. Most of them will require adaptation of the surrounding context. Right now, the goal is to make sure the invocations are syntactically correct.